part-2-server/server.js

```diff
@@ -0,0 +1,50 @@
+var http = require('http');
+var fs= require('fs');
+
+var server = http.createServer(function(request, response){
+
+	
+		if(request.url === '/'){
+			fs.readFile('index.html', function(err,data){
+				if(err){
+				console.log(err)
+			} else {
+				response.write(data.toString())
+				response.end()
+				}
+			})
+		} else if (request.url === '/page-2'){
```

Comment:
The instructions said to respond with the `page-2.html` if the request.url was 'page-2' OR 'page-3' so this should be `request.url === '/page-2 || request.url === '/page-3'`